src/citrine/resources/gemd_resource.py

```diff
@@ -50,30 +60,6 @@ def build(self, data: dict) -> DataConcepts:
         """
         return super().build(data)
 
-    def update(self, model: DataConcepts) -> DataConcepts:
-        """Update a data model object using the appropriate collection."""
-        return self._collection_for(model).update(model)
-
-    def delete(self, uid: Union[UUID, str, LinkByUID, DataConcepts], *, dry_run=False):
-        """
-        Delete a GEMD resource from the appropriate collection.
-
-        Parameters
-        ----------
-        uid: Union[UUID, str, LinkByUID, DataConcepts]
-            A representation of the resource to delete (Citrine id, LinkByUID, or the object)
-        dry_run: bool
-            Whether to actually delete the item or run a dry run of the delete operation.
-            Dry run is intended to be used for validation. Default: false
-
-        """
-        model = self.get(uid)  # Get full object for collection lookup
-        return self._collection_for(model).delete(model, dry_run=dry_run)
-
-    def register(self, model: DataConcepts, *, dry_run=False) -> DataConcepts:
-        """Register a GEMD object to the appropriate collection."""
-        return self._collection_for(model).register(model, dry_run=dry_run)
-
```

Comment:
These are now naturally dispatched through DataConcepts.  You get a `storables` route iff you ask for it.